website/docs/jellyfish/api/spv.md

```diff
@@ -181,3 +181,30 @@ interface ListHtlcsOutputsResult {
   spent: SpentInfo
 }
 ```
+
+## createAnchor
+
+
```

Comment:
Should have `Create, sign and send anchor tx, using only spv api` as comment like what you wrote in the rpc